src/main.cpp

```diff
@@ -2991,10 +2991,7 @@ void vcMain_ShowLoginWindow(vcState *pProgramState)
     {
       ImGui::TextUnformatted(vcString::Get(loginStatusKeys[pProgramState->loginStatus]));
```

Comment:
Sorry my bad, during a rebase I accidentally removed some code.

It should be this:

      const bool isErrorStatus = udStrBeginsWith(loginStatusKeys[pProgramState->loginStatus], "loginError");

      if (isErrorStatus)
        ImGui::PushStyleColor(ImGuiCol_Text, ImVec4(1.0, 0.0, 0.0, 1.0));

      ImGui::TextUnformatted(vcString::Get(loginStatusKeys[pProgramState->loginStatus]));

      if (isErrorStatus)
        ImGui::PopStyleColor();

Reply:
Fixed